### PR TITLE
Refactoring: Separate module codes into `module` directory

### PR DESCRIPTION
**Pull request by yichoi**
First step to https://github.com/Samsung/iotjs/issues/436 : Make core logic independent of each module code

### CODE REVIEW

File: src/iotjs_binding_helper.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+  JObject jexit(process->GetProperty("emitExit"));
+  IOTJS_ASSERT(jexit.IsFunction());
+
+  JArgList args(1);
+  args.Add(JVal::Number(code));
+
+  JResult jres = jexit.Call(JObject::Undefined(), args);
+  if (!jres.IsOk()) {
+    exit(2);
```
Comment (LaszloLango) [inline src/iotjs_binding_helper.cpp]: It would be better to use defines for exit codes.

Comment (akosthekiss) [inline src/iotjs_binding_helper.cpp]: As this is more of a code reorg patch, it's OK to keep original approaches IMO. (That said, I'd second a change in the future.)

File: src/iotjs_binding_helper.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```
Comment (akosthekiss) [inline src/iotjs_binding_helper.cpp]: As this code is originating from existing sources, better keep 2015-2016 copyright years

File: src/iotjs_binding_helper.h [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2015 Samsung Electronics Co., Ltd.
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```
Comment (akosthekiss) [inline src/iotjs_binding_helper.h]: Same here: copyright 2015-2016

File: src/module/iotjs_module_process.h [hunk 4]
```diff
@@ -0,0 +1,30 @@
+/* Copyright 2015 Samsung Electronics Co., Ltd.
```
Comment (akosthekiss) [inline src/module/iotjs_module_process.h]: 2015-2016

**Comment by yichoi** [comment]
@akosthekiss I've modified notices as per your comment

**Comment by yichoi** [comment]
@LaszloLango Thanks for your comment. I will make another PR separately to reflect your comment.

**Comment by yichoi** [comment]
@akosthekiss 2nd try :)

**Pull request merged**